scripts/provisioning/setup.yaml

```diff
@@ -36,6 +36,9 @@ s3:
   reset:
     cmd: /opt/seagate/cortx/s3/bin/s3_setup reset
     args: --config $URL
+  prepare:
+    cmd: /opt/seagate/cortx/s3/bin/s3_setup prepare
+    args: --config $URL
```

Comment:
I think we should put 'null' here as we do not use config option in prepare. Same we can do for post_install. Can you please check with Provisioner team (@nitin-seagate  or @ypise ) if we can use 'null' value ?